elasticapm/traces.py

```diff
@@ -511,7 +537,15 @@ def __init__(
         self.extra = extra
         self.skip_frames = skip_frames
         self.leaf = leaf
-        self.tags = tags
+        if tags and not labels:
```

Comment:
if a users forgets `s/tags/labels/` in one call site; will they have both `tags` and `labels` here and therefore get `tags` silently dropped?
is this behaviour consistent with other agents?

Reply:
No, this would only be an issue if the user uses both `tags` and `labels` in the same `capture_span` call, e.g.

    @capture_span(tags={"foo": "bar"}, labels={"bar": "baz"})
    def my_func():
        pass

That seemed like a silly enough case to not bother handling it.